client/src/pages/Login/Login.tsx

```diff
@@ -40,6 +41,16 @@ function Login() {
       });
   };
 
+  const webMockLogin = () => {
```

Comment:
should this cal login is debugUSerNAme is falsy?

Reply:
you mean when `debugUsername == false` with automatic type conversions? I'm only mocking the login for purposes of testing so this case should not occur.